bundles/net.tourbook/src/net/tourbook/ui/tourChart/SlideoutTourChartOptions.java

```diff
@@ -265,6 +269,27 @@ private void createUI_20_Controls(final Composite parent) {
 
             _chkShowSrtmData.addSelectionListener(_defaultSelectionListener);
          }
+         {
+            /*
+             * label: Night Sections Opacity
+             */
+            _chkShowNightSections = new Button(container, SWT.CHECK);
+            _chkShowNightSections.setText(Messages.Slideout_TourChartOptions_Check_NightSectionsOpacity);
+            _chkShowNightSections.setToolTipText(Messages.Slideout_TourChartOptions_Check_NightSectionsOpacity_Tooltip);
+            _chkShowNightSections.addSelectionListener(_defaultSelectionListener);
+
+            /*
+             * Night Sections Opacity Scale
+             */
+            _spinnerNightSectionsOpacity = new Spinner(container, SWT.BORDER);
```

Comment:
When this slideout is opened and the night section is not selected, the spinner is enabled, it only get's disabled when the checkbox is modified

![mt night section](https://user-images.githubusercontent.com/1283445/115882397-01871880-a44d-11eb-800e-488e258689c7.png)

Reply:
I can't reproduce, it seems to work fine for me. Were you using my latest code when you tested ?

![image](https://user-images.githubusercontent.com/25235665/116118409-830fbe00-a67a-11eb-94f1-93764224af23.png)

Reply:
> 
> 
> I can't reproduce, it seems to work fine for me. Were you using my latest code when you tested ?
> 
> ![image](https://user-images.githubusercontent.com/25235665/116118409-830fbe00-a67a-11eb-94f1-93764224af23.png)

Im using the main branch which is merged with your last PR, fixed in https://github.com/wolfgang-ch/mytourbook/commit/213fbc640206f1c29dbc68863d1a87a0f3c547fb

When I checked this issue on Linux, it was also not disabled when the slideout was opened but then I got this error

![mt opacity too big](https://user-images.githubusercontent.com/1283445/116238206-60a4a000-a761-11eb-8dd4-d87e9ef9fbfe.png)

I think the method which returns the opacity value should check the bounds. The reason for the error could be that an old pref value was too large before changing to 0...100

Reply:
Not sure how to get the value 303.... but I have bounded the returned value [here](https://github.com/FJBDev/mytourbook/pull/57/commits/2396c45839e3468fe509e0664d3af3c2e169b3bc)